cloudwatch-exporter.yaml

```diff
@@ -27,7 +27,9 @@ pipeline:
       expected-commit: af4e8aea04a02c858c19e2ddbaa2cd87ef46e971
 
   - runs: |
-      mvn package
+      # This change is being introduced in https://github.com/prometheus/cloudwatch_exporter/pull/567 (still unmerged)
+      # Check if the version bump below is still needed next time this package is updated
```

Comment:
nit: This is a helpful comment — but just an FYI, normally the updater bot only changes the `version:` line above (and sometimes a SHA update somewhere), and that means only those lines show in the GitHub diff when someone reviews the automated update PR.

For comments that should be checked on version bumps, it's really great to have those comments literally on the same line as `version:`, because it greatly increases the odds that a reviewer will notice the comment when approving an update PR!

Reply:
(The first part of the comment still makes sense here in the pipeline, I think! I'm just talking about the "next time this package is updated" part!)

Reply:
Ohh I see. I'll move the comment up.